music21/musicxml/m21ToXml.py

```diff
@@ -6167,8 +6165,7 @@ def codaToXml(self, coda: repeat.Coda) -> Element:
         >>> MEX.dump(mxCodaText)
         <direction>
           <direction-type>
-            <words default-y="20" enclosure="none"
-                justify="center">Coda</words>
+            <words default-y="20" justify="center">Coda</words>
```

Comment:
Maybe, as a test that XML `enclosure="none"` output is still working, you could set this object (and just this Coda object) to give an explicit no enclosure?

```
>>> c.enclosure = Enclosure.NO_ENCLOSURE
```

and then see that the old-output is still possible?  I'm not seeing any tests of the musicxml output of NO_ENCLOSURE.

Thanks!

Reply:
Done.